pkgs/top-level/linux-kernels.nix

```diff
@@ -865,6 +865,9 @@ in
         ++ lib.optionals (lib.versionAtLeast version "4.16") [
           buildPackages.bison
           buildPackages.flex
+        ]
+        ++ lib.optionals (lib.versionAtLeast version "5.10") [
+          buildPackages.pahole
```

Comment:
Can you explain this addition?

Reply:
Since 5.18 (typo, will correct), pahole is an optional dependency for building a kernel config: https://cateee.net/lkddb/web-lkddb/PAHOLE_VERSION.html

Reply:
Should that go in a separate commit with an explanatory commit message then?